### PR TITLE
GUVNOR-3048: [GDT Column Wizard] No Operator option is assumed as invalid

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/GUVNOR-3048

asaas

### CODE REVIEW

**Comment by karreiro** [comment]
@manstis @jomarko Could you please take a look?

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/OperatorPageTest.java [hunk 1]
```diff
@@ -324,7 +338,7 @@ private void spyOperatorsDropdown() {
 
             return spy(new CEPOperatorsDropdown(operators,
                                                 editingCol));
-        }).when(page).newCepOperatorsDropdown(any());
+        }).when(page).newCepOperatorsDropdown(any(), any());
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/OperatorPageTest.java]: You've reformatted the classes; so you may as well ensure your changes are correctly formatted too! ;-)

Comment (karreiro) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/OperatorPageTest.java]: Good catch! Fixed ;-)

**Comment by karreiro** [comment]
@manstis Thanks. I'm going to double check the builds as we merge the others.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
@manstis Could you approve this one? :-)

**Pull request merged**